src/BLE_Client.cpp

```diff
@@ -20,20 +20,19 @@ appearance: 1156, manufacturer data: 640302018743, serviceUUID:
 #include <memory>
 #include <NimBLEDevice.h>
 
-int reconnectTries = MAX_RECONNECT_TRIES;
-int scanRetries    = MAX_SCAN_RETRIES;
-
 TaskHandle_t BLEClientTask;
 
 SpinBLEClient spinBLEClient;
 
+static MyClientCallback myClientCallback;
+
 void SpinBLEClient::start() {
   // Create the task for the BLE Client loop
   xTaskCreatePinnedToCore(bleClientTask,   /* Task function. */
                           "BLEClientTask", /* name of task. */
                           4700,            /* Stack size of task */
                           NULL,            /* parameter of the task */
-                          1,               /* priority of the task  */
+                          0,               /* priority of the task  */
```

Comment:
I don't think you want to change this to idle priority, suggest leaving it as 1

Reply:
Good call - that's a testing leftover.